File: 351001/shevchenkoAD/DistComp3/Publisher.Domain/Entities/Label.cs
using System.ComponentModel.DataAnnotations.Schema;
using Publisher.Domain.Abstractions;

namespace Publisher.Domain.Entities;

[Table("tbl_label")]
public class Label : BaseEntity
{
    [Column("name")]
    public string Name { get; set; } = null!;
    
    public virtual ICollection<Issue> Issues { get; set; } = new List<Issue>();
}

File: 351002/Alexeichik/MyStartRestApplication/publisher/publisher-impl/src/main/java/com/github/Lexya06/startrestapp/publisher/impl/model/entity/realization/Label.java
package com.github.Lexya06.startrestapp.publisher.impl.model.entity.realization;

import com.github.Lexya06.startrestapp.publisher.impl.model.entity.abstraction.BaseEntity;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.util.Set;
import java.util.HashSet;

@Getter
@Setter
@Entity
@Table(name = "tbl_label")
public class Label extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "labels_seq")
    @SequenceGenerator(name = "labels_seq", sequenceName = "labels_seq", allocationSize = 1)
    private Long id;

    @Column(length = 32, unique = true)
    private String name;

    @ManyToMany(mappedBy = "labels", fetch = FetchType.LAZY)
    private Set<Article> articles = new HashSet<>();
}
